recipes/krb5/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.14.2" %}
+
+package:
+  name: krb5
+  version: {{ version }}
+
+source:
+  fn: krb5-{{ version }}.tar.gz
+  url: https://github.com/krb5/krb5/archive/krb5-1.14.2-final.tar.gz
+  sha256: e8ee5336ee3e9f5a4a9696ab12010a7bb105641788edd95323aeab3e77825b1a
+  patches:
+    - gawk.patch  # [win]
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - bison  # [not win]
+    - autoconf  # [not win]
+    - perl >=5.20.3.1
+    - python <3  # Test only.
```

Comment:
Should this be a skip thing? Also, we need `python` for Windows or are we going to try the `vc` package.